test/test_pipe.py

```diff
@@ -69,7 +69,7 @@ def test_fast_protein_package(self):
             n.write(inseqs)
             n.flush()
 
-            cmd = "%s pipe --sequences %s --otu_table /dev/stdout --singlem_packages %s" % (
+            cmd = "%s pipe --include-off-target-hits --sequences %s --otu_table /dev/stdout --singlem_packages %s" % (
```

Comment:
Thanks for laboriously fixing these tests. But:
* I think fixing the package instead of the cmd would be better, since these methods are for testing the default pipe invocation, and I'm wary of testing become less like "normal" operation, if you see what I mean.
* There's 2 test functions that are deleted - how come?

Reply:
Fair enough.

The test functions that I deleted were exact duplicates to existing ones, they were showing up in Spyder so I thought that they were a bit unnecessary.